src/status_im/ui/screens/db.cljs

```diff
@@ -160,6 +160,8 @@
                   :chat/raw-unviewed-messages
                   :chat/bot-db
                   :chat/geolocation
+                  :chat/message-preview-waiting
+                  :profile/profile-edit
```

Comment:
why did you add :profile/profile-edit?

Reply:
Hm, I didn't add this, I think this happened when I've rebased to develop. I don't know why it's reported as my change